ui/Comments/CommentBlock.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { createRef } from "react";
+import { View, FlatList, TextInput, StyleSheet } from "react-native";
+import { Overlay } from "react-native-elements";
+
+import MaximisedComment from "./MaximisedComment";
+import MainComment from "./MainComment";
+import ReplyBox from "./ReplyBox";
+import BackButton from "./BackButton";
+
+type Props = {
+  onBack: () => void;
+  comment: IComment;
+};
+
+const CommentBox = (props: Props) => {
+  let inpRef: React.RefObject<TextInput> = createRef();
+
+  const focusInputBox = () => {
+    if (inpRef.current) inpRef.current.focus();
+  };
+
+  const onReplyHandler = () => {
+    focusInputBox();
+  };
+
+  return (
+    <Overlay
+      isVisible={true}
+      fullScreen={true}
```

Comment:
tip: instead of `isVisible={true}`, you can simply write `isVisible` - same for `fullScreen`. Just syntactic sugar.